chat-plugins/wtp.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+/********************
+ * Who's that Pokemon?
+********************/
+const cards = require('../config/card-data.js');
+
+
+const maxMistakes = 6;
+
+class Wtp {
+	constructor(room, user, mon, hint1, hint2, hint3) {
+		this.room = room;
+		this.creator = user.userid;
+		this.mon = mon;
+		this.hint1 = Tools.escapeHTML(hint1);
+		this.hint2 = Tools.escapeHTML(hint2);
+		this.hint3 = Tools.escapeHTML(hint3);
+		this.guesses = [];
+	}
+
+	initDisplay() {
+		this.room.addRaw('<div class="sentence-container"><font size="4">Who\'s That Pokemon has been started by ' + this.creator + '<br />Hints: <font color="red">' + this.hint1 + ' - ' + this.hint2 + ' - ' + this.hint3 + '</font></font></div>');
+	}
+
+	guess(guess, user) {
+		let guessed = toId(guess);
+		if (user.userid === this.creator) return user.sendTo(this.room, "You can't guess in your own wtp game.");
```

Comment:
this check can by bypassed if the creator changes their name, which is why it'd be better if you made this.creator hold a reference to the user object instead